src/run.py

```diff
@@ -59,4 +57,4 @@ def send_message(self, chat_id, text, reply_markup=None, emojize=True):
 if __name__ == '__main__':
     logger.info('Bot started')
     nashenas_bot = Bot(telebot=bot)
-    nashenas_bot.run()
```

Comment:
This is not right. You're just deleting the run part.

Reply:
that part was extra. we add bot.infinity_pulling in init and we do not have run method in our class. out bot start running when initialized

Reply:
I see. So `nashenas_bot` name should be fixed cause this is a template bot. Can you change it to 

```
logger.info('Bot started...')
Bot(telebot=bot)
```

Reply:
yea that's right. I will do that